### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by okwme**
https://github.com/cosmos/gaia/issues/1276

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,15 +40,14 @@ gzip -d genesis.cosmoshub-4.json.gz
...
+sed -i '' 's/minimum-gas-prices = ""/minimum-gas-prices = "0.001uatom"/' $HOME/.gaia/config/app.toml
+sed -i '' 's/persistent_peers = ""/persistent_peers = "6e08b23315a9f0e1b23c7ed847934f7d6f848c8b@165.232.156.86:26656,ee27245d88c632a556cf72cc7f3587380c09b469@45.79.249.253:26656,538ebe0086f0f5e9ca922dae0462cc87e22f0a50@34.122.34.67:26656,d3209b9f88eec64f10555a11ecbf797bb0fa29f4@34.125.169.233:26656,bdc2c3d410ca7731411b7e46a252012323fbbf37@34.83.209.166:26656,585794737e6b318957088e645e17c0669f3b11fc@54.160.123.34:26656,5b4ed476e01c49b23851258d867cc0cfc0c10e58@206.189.4.227:26656"/' $HOME/.gaia/config/config.toml
 
 # Configure State sync
-cd $HOME/.gaia/config
-sed -i 's/enable = false/enable = true/' config.toml
-sed -i 's/trust_height = 0/trust_height = <BLOCK_HEIGHT>/' config.toml
-sed -i 's/trust_hash = ""/trust_hash = "<BLOCK_HASH>"/' config.toml
-sed -i 's/rpc_servers = ""/rpc_servers = "https:\/\/rpc.cosmos.network:443,https:\/\/rpc.cosmos.network:443"/' config.toml
+sed -i '' 's/enable = false/enable = true/' $HOME/.gaia/config/config.toml
```
Comment (glnro) [inline README.md:47]: getting an error when I run this on my linux box

**Comment by glnro** [comment]
I think we need to keep both...and add a comment for mac and linux.....or just create a separate tab for each type of machine....wdyt?

**Comment by okwme** [comment]
oh interesting....
did you try it woutout the space after the `-i` ?
https://stackoverflow.com/a/14813278/433178

**Comment by glnro** [comment]
> oh interesting.... did you try it woutout the space after the `-i` ? https://stackoverflow.com/a/14813278/433178

yeah that works 👍

File: README.md [hunk 2]
```diff
@@ -40,15 +40,14 @@ gzip -d genesis.cosmoshub-4.json.gz
 mv genesis.cosmoshub-4.json $HOME/.gaia/config/genesis.json
 
 #Set minimum gas price & peers
-sed -i 's/minimum-gas-prices = ""/minimum-gas-prices = "0.001uatom"/' app.toml
-sed -i 's/persistent_peers = ""/persistent_peers = "6e08b23315a9f0e1b23c7ed847934f7d6f848c8b@165.232.156.86:26656,ee27245d88c632a556cf72cc7f3587380c09b469@45.79.249.253:26656,538ebe0086f0f5e9ca922dae0462cc87e22f0a50@34.122.34.67:26656,d3209b9f88eec64f10555a11ecbf797bb0fa29f4@34.125.169.233:26656,bdc2c3d410ca7731411b7e46a252012323fbbf37@34.83.209.166:26656,585794737e6b318957088e645e17c0669f3b11fc@54.160.123.34:26656,5b4ed476e01c49b23851258d867cc0cfc0c10e58@206.189.4.227:26656"/' config.toml
+sed -i '' 's/minimum-gas-prices = ""/minimum-gas-prices = "0.001uatom"/' $HOME/.gaia/config/app.toml
```
Comment (okwme) [inline README.md:43]:
```suggestion
sed -i'' 's/minimum-gas-prices = ""/minimum-gas-prices = "0.001uatom"/' $HOME/.gaia/config/app.toml
```

File: README.md [hunk 3]
```diff
@@ -40,15 +40,14 @@ gzip -d genesis.cosmoshub-4.json.gz
...
+sed -i '' 's/persistent_peers = ""/persistent_peers = "6e08b23315a9f0e1b23c7ed847934f7d6f848c8b@165.232.156.86:26656,ee27245d88c632a556cf72cc7f3587380c09b469@45.79.249.253:26656,538ebe0086f0f5e9ca922dae0462cc87e22f0a50@34.122.34.67:26656,d3209b9f88eec64f10555a11ecbf797bb0fa29f4@34.125.169.233:26656,bdc2c3d410ca7731411b7e46a252012323fbbf37@34.83.209.166:26656,585794737e6b318957088e645e17c0669f3b11fc@54.160.123.34:26656,5b4ed476e01c49b23851258d867cc0cfc0c10e58@206.189.4.227:26656"/' $HOME/.gaia/config/config.toml
```
Comment (okwme) [inline README.md:44]:
```suggestion
sed -i'' 's/persistent_peers = ""/persistent_peers = "6e08b23315a9f0e1b23c7ed847934f7d6f848c8b@165.232.156.86:26656,ee27245d88c632a556cf72cc7f3587380c09b469@45.79.249.253:26656,538ebe0086f0f5e9ca922dae0462cc87e22f0a50@34.122.34.67:26656,d3209b9f88eec64f10555a11ecbf797bb0fa29f4@34.125.169.233:26656,bdc2c3d410ca7731411b7e46a252012323fbbf37@34.83.209.166:26656,585794737e6b318957088e645e17c0669f3b11fc@54.160.123.34:26656,5b4ed476e01c49b23851258d867cc0cfc0c10e58@206.189.4.227:26656"/' $HOME/.gaia/config/config.toml
```

File: README.md (hunk 1 above)
Comment (okwme) [inline README.md:47]:
```suggestion
sed -i'' 's/enable = false/enable = true/' $HOME/.gaia/config/config.toml
```

File: README.md [hunk 4]
```diff
@@ -40,15 +40,14 @@ gzip -d genesis.cosmoshub-4.json.gz
...
 
 # Configure State sync
-cd $HOME/.gaia/config
-sed -i 's/enable = false/enable = true/' config.toml
-sed -i 's/trust_height = 0/trust_height = <BLOCK_HEIGHT>/' config.toml
-sed -i 's/trust_hash = ""/trust_hash = "<BLOCK_HASH>"/' config.toml
-sed -i 's/rpc_servers = ""/rpc_servers = "https:\/\/rpc.cosmos.network:443,https:\/\/rpc.cosmos.network:443"/' config.toml
+sed -i '' 's/enable = false/enable = true/' $HOME/.gaia/config/config.toml
+sed -i '' 's/trust_height = 0/trust_height = <BLOCK_HEIGHT>/' $HOME/.gaia/config/config.toml
```
Comment (okwme) [inline README.md:48]:
```suggestion
sed -i'' 's/trust_height = 0/trust_height = <BLOCK_HEIGHT>/' $HOME/.gaia/config/config.toml
```

File: README.md [hunk 5]
```diff
@@ -40,15 +40,14 @@ gzip -d genesis.cosmoshub-4.json.gz
...
+sed -i '' 's/trust_hash = ""/trust_hash = "<BLOCK_HASH>"/' $HOME/.gaia/config/config.toml
```
Comment (okwme) [inline README.md:49]:
```suggestion
sed -i'' 's/trust_hash = ""/trust_hash = "<BLOCK_HASH>"/' $HOME/.gaia/config/config.toml
```

File: README.md [hunk 6]
```diff
@@ -40,15 +40,14 @@ gzip -d genesis.cosmoshub-4.json.gz
...
+sed -i '' 's/rpc_servers = ""/rpc_servers = "https:\/\/rpc.cosmos.network:443,https:\/\/rpc.cosmos.network:443"/' $HOME/.gaia/config/config.toml
```
Comment (okwme) [inline README.md:50]:
```suggestion
sed -i'' 's/rpc_servers = ""/rpc_servers = "https:\/\/rpc.cosmos.network:443,https:\/\/rpc.cosmos.network:443"/' $HOME/.gaia/config/config.toml
```

**Comment by faddat** [comment]
This is using sdk 46, so we should probably close it.

**Comment by mmulji-ic** [comment]
Outdated now

**Pull request closed without merging**